cves/kernel/CVE-2016-5728.yml

```diff
@@ -147,11 +152,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
-autodiscoverable:
+  answer: |
```

Comment:
Who is them? The developers or an outside source? If it's the developers the `developer` field should probably be updated to true.